app/(afterLogin)/(okrplanning)/okr/_components/keyresultForm/numericForm/index.tsx

```
@@ -222,10 +222,15 @@ const NumericForm: React.FC<OKRFormProps> = ({
                   { required: true, message: 'Please enter the target value' },
                   {
-                    validator: (form, value) =>
-                      value && value >= 0
-                        ? Promise.resolve()
-                        : Promise.reject('Target value must be non-negative'),
+                    validator: (form, value) => {
+                      if (value <= 0) {
+                        return Promise.reject('Target must be greater than 0');
+                      }
+                      if (value <= keyItem.initialValue) {
+                        return Promise.reject('Target must be greater than initial value');
```

ESLint

[error] Replace `'Target·must·be·greater·than·initial·value'` with `⏎··························'Target·must·be·greater·than·initial·value',⏎························`

(prettier/prettier)